content/javascript/concepts/strings/terms/to-string/to-string.md

```diff
@@ -0,0 +1,48 @@
+
+Title: '.toString()'
+Description: 'Returns a string representing the object'
+Subject: 
+ - 'Computer Science'
+Tags:
+ -  'Functions'
+ -  'Strings'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+
+The **`.toString()`** method converts the given object and returns its corresponding string representation.
+
+ ## Syntax
+
+ ```js
+ .toString()
+ ```
```

Comment:
```suggestion
## Syntax

```pseudo
stringObject.toString()
```
```

Removing the extra spacing in front for these lines